test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -689,7 +686,41 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
     });
 
     Expect.describe('Device Fingerprint', function () {
-      itp('does not contain device fingerprint header in primaryAuth if feature is enabled', function () {
+      itp('contains fingerprint header in get security image request if feature is enabled', function () {
```

Comment:
Looks like these two tests will apply to IDPDiscovery as well. Should we add them to IDPDiscovery_spec also?

Reply:
Good point, I'll add them 👍